webpack.config.js

```diff
@@ -88,14 +91,26 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                     test: /\.css/,
                     exclude: /ckeditor5-[^/\\]+[/\\]theme[/\\].+\.css$/,
                     use: [
-                        MiniCssExtractPlugin.loader,
+                        {
+                            loader: MiniCssExtractPlugin.loader,
+                            options: {
+                                // https://github.com/webpack-contrib/mini-css-extract-plugin/issues/44#issuecomment-391009221
+                                publicPath: '../../',
```

Comment:
this looks false in this webpack config file

Reply:
why do you think its wrong? i tried to explain whats happening here: https://github.com/sulu/sulu/pull/6225/files#r708199344

Reply:
I think this will not longer make it possible to overwrite the public directory. Not tested it but I think using  `path.resolve(projectRootPath, publicDir)` the configured directory like above should also end in the correct result?

Reply:
i dont think this is related to the public directory 🙂 
this config only affects the path in the bundled css for resources that are referenced inside of `.scss` files. i am not sure how to explain it better than in https://github.com/sulu/sulu/pull/6225#discussion_r708209437 - maybe we should talk about it in person?

Reply:
Where does then the `../../` come from when I configured sulu into another directory and so have example has 3 directories to the publicDir: `build/another-directory/admin` then I think the `../../../` should be here? Do we need this in the general webpack config can we introduce instead a parameter for styleguidist directory?

Reply:
yes, if you set `env.output_path` to `build/another-directory/admin`, we need to use `../../../`. i changed the code to `path.relative(outputPath, '.')` to do this automatically 🙂

Reply:
> Do we need this in the general webpack config can we introduce instead a parameter for styleguidist specific things or overwrite it in there configuration.

as long as you are not using sulu inside of a subfolder, hardcoding `/` here (like it was done before: https://github.com/sulu/sulu/pull/6225/files#r708207169) works. if we want to allow running sulu in a subfolder, we should define it like this

Reply:
Looks then good for me.